sycl/plugins/level_zero/tracing.cpp

```diff
@@ -30,6 +30,8 @@ constexpr int GMinVer = 1;
 #ifdef XPTI_ENABLE_INSTRUMENTATION
 static xpti_td *GCallEvent = nullptr;
 static xpti_td *GDebugEvent = nullptr;
+static uint8_t GCallStreamID = 0;
+static uint8_t GDebugStreamID = 0;
```

Comment:
Is there a data race here for multi-threaded apps?

Reply:
Shouldn't be a data race as the initialization should be done once in piPluginInit() where zeEnableTracing() is called and the only place it is called from.